tfhe/src/core_crypto/commons/generators/encryption/mod.rs

```diff
@@ -297,8 +294,7 @@ impl<G: ByteRandomGenerator> EncryptionRandomGenerator<G> {
     ) where
         (Scalar, Scalar): RandomGenerable<Gaussian<f64>>,
     {
-        self.noise
-            .fill_slice_with_random_gaussian(output, 0., std.get_standard_dev());
+        self.noise.fill_slice_with_random_noise(output, std)
```

Comment:
I would rename `fill_slice_with_random_noise` to something like `fill_slice_with_(random)_gaussian_(noise)`
Same comment for `random_noise_custom_mod` to `(random)_gaussian_(noise)_custom_mod`

Having a name meaning that the mean is 0 would be even better but I don't have an idea of such a short name

Reply:
I re-used the previous EncryptionRandomGenerator notation as it now "just" wraps a mask and noise generator

Reply:
but I can see your point

Reply:
I'll think about it as for now our noise are always gaussian as far as the user is concerned

Reply:
Issue https://github.com/zama-ai/tfhe-rs-internal/issues/222